plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatement.java

```diff
@@ -247,6 +253,47 @@ public void createTextEdit(ComputedInfo computedInfo) {
         }
     }
 
+    // function to get the amount of commas at the end of import string
+    private int verifyComma(String line) {
+        // TODO Auto-generated method stub
+        boolean withComma = true;
+        int commaAmount = 0;
+        int len = line.length();
+
+        if (line.endsWith(")")) {
+
+            line = line.substring(0, len - 1);
+            line = line.trim();
+            len = line.length();
+
+            while (withComma == true) {
+                if (line.endsWith(",")) {
+                    line = line.substring(0, len - 1);
+                    len--;
+                    commaAmount++;
+                } else {
+                    withComma = false;
+                    line = line.concat(")");
+                    len++;
+                }
+            }
+
+        } else {
+
+            while (withComma == true) {
```

Comment:
I don't think you need the `withComma` variable at all... you can just do: `while(line.endswith(',')`. Also, this `while` is equal to the `while` in the block above, you can just have a single `while` out of the else and remove  the previous one.